ethereum/Deploying.md

```diff
@@ -8,35 +8,11 @@ This is the deployment process:
 # The Secret Recovery Phrase for the wallet the contract will be deployed from.
 export MNEMONIC=...
 
-# Set the deploy commit hash in the contract (used for debugging purposes)
-sed -i "s/__DEPLOY_COMMIT_HASH_PLACEHOLER__/$(git rev-parse HEAD)/g" ./contracts/pyth/Pyth.sol
-
-# Ensure that we deploy a fresh build with up-to-date dependencies.
-rm -rf build && npx truffle compile --all
-
-# Merge the network addresses into the artifacts, if some contracts are already deployed.
-npx apply-registry
-
-# After doing the above steps, you can run the below commands per each network.
-
-# Load the configuration environment variables for deploying your network. make sure to use right env file.
-# If it is a new chain you are deploying to, create a new env file and commit it to the repo.
-rm -f .env; ln -s .env.prod.xyz .env && set -o allexport && source .env set && set +o allexport
-
-# Perform the migration step by step using `--to <migration file number>` argument. Some steps require a governance execution to be successful. //TODO the process.
-# You might need to repeat the steps because of busy RPCs.
-# Also, sometimes the gases are not adjusted. Please update them with the network
-# explorer gas tracker.
-# Tips in Troubleshooting section below can help in case of any error.
-npx truffle migrate --network $MIGRATIONS_NETWORK --to <step>
-
-# Some steps require executing a governance instruction to be successful, you can use the multisig message builder tool in 
-# `third_party/pyth` of this repo root to create multisig transaction and execute it to create the VAA.
-# Then you can use the VAA (in hex) to execute the governance instruction. To do so, run:
-$ npx apply-registry # apparently a deployProxyImpl messes up with correct address of proxy
-$ npx truffle console --network $MIGRATIONS_NETWORK
-> let p = await PythUpgradable.deployed()
-> await p.executeGovernanceInstruction("<VAA in hex like: 0x123002342352>");
+# Perform the migration
+# You might need to repeat this script because of busy RPCs. Repeating would not cause any problem even
+# if the changes are already made. Also, sometimes the gases are not adjusted. Please update them with
+# the network explorer gas tracker. Tips in Troubleshooting section below can help in case of any error.
+./deploy.sh <network_a> <network_b> <...>
```

Comment:
Could we add some examples of how to call this script with concrete values here (in comments)? A couple of testnet networks, and a couple of mainnet networks.